blueprints/stirling/docker-compose.yml

```diff
@@ -1,22 +1,15 @@
+version: '3.3'
 services:
   stirling-pdf:
-    image: frooodle/s-pdf:latest
+    image: docker.stirlingpdf.com/stirlingtools/stirling-pdf:latest
     ports:
-      - 8080
+      - '8080:8080'
     volumes:
-      - stirling_pdf_trainingdata:/usr/share/tessdata
-      - stirling_pdf_extraconfigs:/configs
-      - stirling_pdf_customfiles:/customFiles/
-      - stirling_pdf_logs:/logs/
-      - stirling_pdf_pipeline:/pipeline/
+      - ./StirlingPDF/trainingData:/usr/share/tessdata # Required for extra OCR languages
+      - ./StirlingPDF/extraConfigs:/configs
+      - ./StirlingPDF/customFiles:/customFiles/
+      - ./StirlingPDF/logs:/logs/
+      - ./StirlingPDF/pipeline:/pipeline/
```

Comment:
we want to stick with data volumes over host mounts because they are more portable.